packages/viewer/src/lib/volume-intersection/controller.ts

```diff
@@ -67,11 +73,14 @@ export class VolumeIntersectionQueryController {
         console.error('Failed to perform volume intersection query', e);
         throw e;
       } finally {
-        this.inFlightOperations = this.inFlightOperations - 1;
+        this.viewer.cameraControls = this.previousViewerCameraControls ?? true;
+        this.previousViewerCameraControls = undefined;
+        this.operationInFlight = false;
+        this.executeComplete.emit();
       }
-    } else if (this.inFlightOperations >= this.maxInFlightOperations) {
+    } else if (this.operationInFlight) {
       throw new Error(
-        `Unable to perform volume intersection query due to the limit of ${this.maxInFlightOperations}.`
+        `Unable to perform volume intersection query as there is already one in-flight.`
       );
     }
```

Comment:
not related to this PR, but should we add an `else` here to handle if the screenBounds _are_ null? Log or throw an error or something?

Reply:
Sure, I can take a look at adding that. Probably at least worth an error